translation_service/src/serge_ts_connector.py

```diff
@@ -87,12 +76,18 @@ def copy_serge_po_files(serge_translation_dir, ts_serge_po_dir, ts_inbox):
                     shutil.copy2(serge_file_path, ts_serge_file_path)
                     shutil.copy2(serge_file_path, ts_inbox_file_path)
 
+    # Localize the project and its po files
+    localize_project(ts_inbox, ts_outbox, translation_api, google_key_path)
 
-def copy_outbox_to_serge(serge_translation_dir, ts_outbox):
-    """Copies the localized .po files from ts_outbox into the serge_translation_dir.
+    # Remove all contents from inbox once processed (localized)
+    shutil.rmtree(ts_inbox)
+
+
+def serge_pull_ts(serge_ts, ts_outbox):
+    """Handles the Serge push_ts command. Copies the localized .po files from ts_outbox into serge_ts.
```

Comment:
not sure if this is a typo- is it supposed to be `the serge pull_ts comand`?

Reply:
Yes, that's a typo! Thanks for finding it. I've fixed it now.